check-diff.sh

```diff
@@ -624,6 +624,20 @@ function lint_js_files {
 	fi
 }
 
+function lint_xml_files {
+	if [ ! -s "$TEMP_DIRECTORY/paths-scope-xml" ]; then
+		return
+	fi
+
+	set -e
+
+	echo "## XMLLINT"
+	cd "$LINTING_DIRECTORY"
+	if ! cat "$TEMP_DIRECTORY/paths-scope-xml" | remove_diff_range | xargs xmllint --noout; then
+		exit 1
```

Comment:
BTW, this is fine, but since `set -e` is above, any command that has a non-zero exit code will automatically cause the script to top execution, so this could actually be even more concise as:

```bash
cat "$TEMP_DIRECTORY/paths-scope-xml" | remove_diff_range | xargs xmllint --noout
```